src/app/lib/config/services/marathi.js

```diff
@@ -165,7 +165,7 @@ export const service = {
     footer: {
       trustProjectLink: {
         href: 'https://www.bbc.com/marathi/institutional-50418391',
-        text: 'तुम्ही बीबीसीवर विश्वास का ठेवू शकता',
+        text: 'तुम्ही बीबीसीवर विश्वास ठेवू शकता कारणा',
```

Comment:
That last word seems a bit different than in translations sheet " कारण "

Reply:
It's a little weird, if I Ctrl+F it matches both, even though I can see the difference. Might be something to do with how these languages are implemented I don't quite understand.

Reply:
I can see the same text across the issue description, PR and translation sheet: `तुम्ही बीबीसीवर विश्वास ठेवू शकता कारण`

Reply:
This is most likely to do with how clever the software rendering the text is.  In some languages, two characters when next to one another combine into a single glyph.  This is like the old English a and e combining to æ in words like encyclopædia. So I think it's fine.